employee-rostering-backend/README.adoc

```diff
@@ -1 +1,57 @@
-// TODO Add instructions for running the application and backend
\ No newline at end of file
+= OptaWeb Employee Rostering Backend
+
+image:https://sonarcloud.io/api/project_badges/measure?project=employee-rostering&metric=alert_status[
+"Quality Gate Status", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=reliability_rating[
+"Reliability Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=security_rating[
+"Security Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=sqale_rating[
+"Maintainability Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=ncloc[
+"Lines of Code", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=coverage[
+"Coverage", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+
+image:https://david-dm.org/kiegroup/optaweb-employee-rostering/status.svg?path=optaweb-employee-rostering[
+"dependencies Status", link="https://david-dm.org/kiegroup/optaweb-employee-rostering?path=optaweb-employee-rostering"]
+image:https://david-dm.org/kiegroup/optaweb-employee-rostering/dev-status.svg?path=optaweb-employee-rostering[
+"devDependencies Status", link="https://david-dm.org/kiegroup/optaweb-employee-rostering?path=optaweb-employee-rostering&type=dev"]
+
+== Build and Run with Maven
+
+In the project backend directory, run the following command to build from source:
+
+----
+mvn clean install
+----
+
+To ignore tests during the build, run:
+
+----
+mvn clean install -DskipTests
```

Comment:
I'd immediately give them the -DskipTests command.
People know what it does, and it's a good thing for people who try it out quickly to ignore the tests so they get it to run faster. Only a small portion of them become contributors and those will know to run the tests before submitting a PR (and if they don't, jenkins does it for them anyway).